go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.33.0
 	github.com/stretchr/testify v1.9.0
 	github.com/tailscale/hujson v0.0.0-20221223112325-20486734a56a
-	github.com/tailscale/tailscale-client-go v1.16.0
+	github.com/tailscale/tailscale-client-go v1.16.1-0.20240424180809-5a248d9547d4
```

Comment:
TODO:
- [ ] uprev to actual version once https://github.com/tailscale/tailscale-client-go/pull/78 is through and a new release is cut